src/booking/containers/booking-date/booking-date.component.html

```diff
@@ -1,4 +1,18 @@
-<form [formGroup]="formGroup">
+<ng-container *ngIf="isAnyError()">
```

Comment:
Have you checked for `expressionchangedafterithasbeencheckederror` in the console tab? If there are no errors caused by this it's fine.